### PR TITLE
Rules for missing 'return' and 'param'

### PR DESCRIPTION
**Pull request by wetzlmaier**

### CODE REVIEW

**Review by AlWoSp — CHANGES_REQUESTED** [review]

File: locale/zh-tw/script.lua [hunk 1]
```diff
@@ -114,6 +114,12 @@ DIAG_UNDEFINED_DOC_NAME               =
 '未定義的類型或別名 `{}`。'
 DIAG_UNDEFINED_DOC_PARAM              =
 '指向了未定義的參數 `{}`。'
+DIAG_MISSING_GLOBAL_DOC               =
```
[the remark this answers is not in the document]
Comment (wetzlmaier) [reply locale/zh-tw/script.lua:117]: added comment

File: test/diagnostics/missing-return-doc.lua [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+---@return integer
+function F4()
+    return 1, <!2!>
+end
+
+print(f1())
+]]
+
+TEST [[
```
[the remark this answers is not in the document]
Comment (wetzlmaier) [reply test/diagnostics/missing-return-doc.lua:50]: removed empty tests

File: locale/en-us/setting.lua [hunk 3]
```diff
@@ -375,6 +375,12 @@ config.diagnostics['duplicate-set-field']   =
 'Enable diagnostics for setting the same field in a class more than once.'
 config.diagnostics['invisible']             =
 'Enable diagnostics for accesses to fields which are invisible.'
+config.diagnostics['missing-global-doc']    =
+'Missing documentation for globals.'
```
[the remark this answers is not in the document]
Comment (wetzlmaier) [reply locale/en-us/setting.lua:379]: I have adapted the text according to the three rules

File: locale/en-us/setting.lua [hunk 4]
```diff
@@ -375,6 +375,12 @@ config.diagnostics['duplicate-set-field']   =
...
+config.diagnostics['missing-param-doc']     =
+'Missing @param annotation.'
+config.diagnostics['missing-return-doc']    =
+'Missing @return annotation.'
```
[the remark this answers is not in the document]
Comment (wetzlmaier) [reply locale/en-us/setting.lua:383]: settings.lua = description of the rule
script.lua = linter messages

File: locale/en-us/setting.lua [hunk 5]
```diff
@@ -375,6 +375,12 @@ config.diagnostics['duplicate-set-field']   =
...
 config.diagnostics['missing-parameter']     =
 'Enable diagnostics for function calls where the number of arguments is less than the number of annotated function parameters.'
```
[the remark this answers is not in the document]
Comment (wetzlmaier) [reply locale/en-us/setting.lua:385]: This file is not mine. Should I change the message?

File: locale/en-us/setting.lua [hunk 6]
```diff
@@ -375,6 +375,12 @@ config.diagnostics['duplicate-set-field']   =
 'Enable diagnostics for setting the same field in a class more than once.'
 config.diagnostics['invisible']             =
 'Enable diagnostics for accesses to fields which are invisible.'
+config.diagnostics['missing-global-doc']    =
+'Missing documentation for globals.'
+config.diagnostics['missing-param-doc']     =
+'Missing @param annotation.'
```
[the remark this answers is not in the document]
Comment (wetzlmaier) [reply locale/en-us/setting.lua:381]: setting.lua = description of the rule
name of function/parameter will be displayed (script.lua)

File: locale/en-us/setting.lua (hunk 5 above)
[the remark this answers is not in the document]
Comment (wetzlmaier) [reply locale/en-us/setting.lua:385]: "function calls" is correct

File: locale/en-us/setting.lua [hunk 7]
```diff
@@ -375,6 +375,12 @@ config.diagnostics['duplicate-set-field']   =
 'Enable diagnostics for setting the same field in a class more than once.'
 config.diagnostics['invisible']             =
 'Enable diagnostics for accesses to fields which are invisible.'
+config.diagnostics['missing-global-doc']    =
```
[the remark this answers is not in the document]
Comment (wetzlmaier) [reply locale/en-us/setting.lua:378]: an additional task is ok

**Review by AlWoSp — APPROVED** [review]

**Review by AlWoSp — CHANGES_REQUESTED** [review]

**Review by AlWoSp — APPROVED** [review]

**Pull request merged**